profiler/shakesapp/README.md

```diff
@@ -0,0 +1,32 @@
+# Profiler Shakespeare Application in Go
+
+This application starts a server which will return the number of times a word or
+phrase appears in the works of Shakespeare, and sends queries to that server to
+create load.
+
+Profiler is enabled for this application, and can be used to identify
+opportunities to optimize the server code.
+
+## Running the application
+
+This application should be run with Go 1.14. Once that version of Go is
+installed:
+
+*   Get the application code:
+
+    ```sh
+    git clone https://github.com/GoogleCloudPlatform/golang-samples.git
+    cd golang-samples/getting-started/profiler/shakesapp
+    ```
+
+*   Install dependencies:
+
+    ```sh
+    go get .
+    ```
+
+*   Run the application:
+
+    ```sh
+    go run . -project_id="your-project-id" -version="application-version"
```

Comment:
Specifying project ID should never be required when running on GCP and these instructions should favor that as the default case. So we should never ask user to specify the project ID in any public profiler doc, except the "running outside GCP" doc section.

Reply:
Done.